deepfence_frontend/apps/dashboard/src/features/onboard/components/connectors/clouds/AWSTerraform.tsx

```diff
@@ -0,0 +1,144 @@
+import cx from 'classnames';
+import { memo } from 'react';
+import { HiViewGridAdd } from 'react-icons/hi';
+import { useCopyToClipboard } from 'react-use';
+import { Button, Card, Step, Stepper, Typography } from 'ui-components';
+
+import { CopyToClipboardIcon } from '../../../../../components/CopyToClipboardIcon';
+import { usePageNavigation } from '../../../../../utils/usePageNavigation';
+
+export const AWSTerraform = memo(() => {
+  const [_, copyToClipboard] = useCopyToClipboard();
```

Comment:
add this to copy to clipboard component itself

Reply:
resolved